lib/resty/rediscluster.lua

```diff
@@ -523,7 +523,7 @@ end
 
 function _M.refresh_slots(self)
     local lock, err, elapsed
-    lock, err = resty_lock:new(self.config.dict_name or DEFAULT_SHARED_DICT_NAME, { time_out = 0 })
+    lock, err = resty_lock:new(self.config.dict_name or DEFAULT_SHARED_DICT_NAME, { timeout = 0 })
```

Comment:
I believe this is a breaking change, so let’s set the `timeout` value to 5, to be consistent with using the default value of 5 for passing the fault `time_out`.

Reply:
The original intention is to set `timeout = 5`.

The fix here does change the behaviour, but change it to the correct intention.

The `timeout = 0` let only one to refresh the cache. The others would return immediately and no need to refresh it again. This would improve the perf a bit.

Reply:
If so, we need to update the README.md and note that this is a breaking change.

Reply:
Will do.

Reply:
Added a new section “Unreleased” in readme.